src/librustdoc/config.rs

```diff
@@ -504,8 +504,18 @@ impl Options {
             return Err(1);
         }
 
-        let output =
-            matches.opt_str("o").map(|s| PathBuf::from(&s)).unwrap_or_else(|| PathBuf::from("doc"));
+        let out_dir = matches.opt_str("out-dir").map(|s| PathBuf::from(&s));
+        let output = matches.opt_str("output").map(|s| PathBuf::from(&s));
+        let output = match (out_dir, output) {
+            (Some(_), Some(_)) => {
+                diag.struct_err("cannot use both 'out-dir' and 'output' at once").emit();
```

Comment:
```suggestion
                diag.struct_err("cannot use both 'out-dir' and 'output' at once").note("'output' is a deprecated alias for 'out-dir', so only one of them can be passed").emit();
```

Reply:
Also, should we be using `struct_fatal` here?

Reply:
I don't think so, struct_fatal stops processing immediately. Other parts of rustdoc are already using struct_err: https://github.com/rust-lang/rust/blob/0369ea1751872d769b50ab889d39abd84ad3fba0/src/librustdoc/config.rs#L424-L427

Reply:
Hmm, okay 👍